frontend/src/components/Categories.tsx

```diff
@@ -1,12 +1,18 @@
 import CategoryItem from "./CategoryItem";
 import './Categories.scss'
+import {Category} from "../models/Category";
+import {ITokenConfig} from "../models/Connection";
 
-export default function Categories() {
-
+export default function Categories(props: {categories: Category[], config: ITokenConfig}) {
+    const {categories, config} = props;
```

Comment:
I would destructure the props already in the function declarement. e.g. 
export interface CategoriesProps {
categories: Categorie[],
config: ITokenConfig
}
[...] Categories({categories, config}: CategoriesProps)